### DISCUSSION TITLE
skeleton during user loading

### DISCUSSION THREAD

**Discussion by vladislav-koval** [greatfrontend/greatfrontend#12] [Ideas/Feature Requests]
Hello, I like your platform for front-end engineers, but when I bought a subscription, I got scared when I refreshed the page and saw a button to buy a subscription. Then I realized that it was just that the user information did not load, so in order to improve ux, I would like to suggest using a skeleton or a spinner until the user information has loaded. Either pass this information through cookies or otherwise. 

Best regards to you

**Comment by gfeeng** [comment]
That's a great suggestion! We'll look into improving our loading states.

**Comment by gfeeng** [comment]
This has been done! We use a fade in transition for the top right section of the navbar.
